src/containers/DownloadCsv/index.jsx

```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+
+import fetchCsv from '../../data/actions/courseEnrollments';
+import DownloadCsv from '../../components/DownloadCsv';
+
+const mapStateToProps = state => ({
+  enterpriseId: state.portalConfiguration.enterpriseId,
```

Comment:
I don't think the `enterpriseId` is necessary for the `DownloadCsv` container. The service layer needs it but you can get the enterprise id like [how it's done here](https://github.com/edx/edx-portal/blob/902d80ef9d2fdc99590fadcfb7b3c4223566b2dd/src/data/services/EnterpriseDataApiService.js#L14).